tools/serverpod_cli/test/generator/dart/client_code_generator/class_copy_with_test.dart

```diff
@@ -383,7 +384,8 @@ void main() {
     test('then the clone method is called on field when copying the object.',
         () {
       var sourceCode = copyWithMethod?.body.toSource();
-      expect(sourceCode, '{return Example(map: map ?? this.map.clone());}');
+      expect(sourceCode,
+          '{return Example(map: map ?? this.map.map((key0, value0) => MapEntry(_i1.$cloneTrivialFunctionName(key0), _i1.$cloneTrivialFunctionName(value0))));}');
```

Comment:
Suggestion: Please update the description so that it matches the new test case.

Same comment here as for the other updated tests.

Reply:
Ahh, good catch! 💯